packages/react-testing/src/root.tsx

```diff
@@ -298,7 +298,7 @@ function flatten(
     return [node.memoizedProps as string];
   }
 
-  const props = {...(node.memoizedProps || {})};
+  const props = {...((node.memoizedProps as any) || {})};
```

Comment:
`Spread types may only be created from object types.`
 
Solved by casting to `any` from `unknown` first.